srv/salt/ceph/rescind/tuned/default.sls

```diff
@@ -0,0 +1,3 @@
+uninstall tuned:
+    pkg.removed:
```

Comment:
Should we remove tuned?

Reply:
I think this is fine since we install it.  Also, if the one admin that is using DeepSea to manage their Ceph cluster also created their own profiles, then the init.sls override is there to disable the uninstall of tuned.